module/core/src/Infrastructure/Exception/DenoralizationException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Core\Infrastructure\Exception;
```

Comment:
Exceptions should be placed in application layer -> infrastructure is aware of application - not the other way around